pylxd/models/operation.py

```diff
@@ -59,7 +59,9 @@ def wait(self):
         response = self._client.api.operations[self.id].wait.get()
 
         try:
-            if response.json()['metadata']['status'] == 'Failure':
+            md = response.json()['metadata']
+            if not md: md = response.json()
+            if md['status'] == 'Failure':
```

Comment:
According to the API docs, this isn't correct.  See: https://github.com/lxc/lxd/blob/master/doc/rest-api.md#10operationsuuid